src/test/ui/privacy/privacy-ns2.stderr

```diff
@@ -43,66 +43,54 @@ LL | use foo1::Bar;
    |
 
 error[E0603]: trait `Bar` is private
-  --> $DIR/privacy-ns2.rs:63:15
+  --> $DIR/privacy-ns2.rs:61:15
    |
 LL |     use foo3::Bar;
    |               ^^^ private trait
    |
 note: the trait `Bar` is defined here
-  --> $DIR/privacy-ns2.rs:55:5
+  --> $DIR/privacy-ns2.rs:53:5
    |
 LL |     trait Bar {
    |     ^^^^^^^^^
 
 error[E0603]: trait `Bar` is private
-  --> $DIR/privacy-ns2.rs:67:15
+  --> $DIR/privacy-ns2.rs:65:15
    |
 LL |     use foo3::Bar;
    |               ^^^ private trait
    |
 note: the trait `Bar` is defined here
-  --> $DIR/privacy-ns2.rs:55:5
+  --> $DIR/privacy-ns2.rs:53:5
    |
 LL |     trait Bar {
    |     ^^^^^^^^^
 
 error[E0603]: trait `Bar` is private
-  --> $DIR/privacy-ns2.rs:74:16
+  --> $DIR/privacy-ns2.rs:72:16
    |
 LL |     use foo3::{Bar,Baz};
    |                ^^^ private trait
    |
 note: the trait `Bar` is defined here
-  --> $DIR/privacy-ns2.rs:55:5
+  --> $DIR/privacy-ns2.rs:53:5
    |
 LL |     trait Bar {
    |     ^^^^^^^^^
 
-error[E0107]: wrong number of const arguments: expected 0, found 1
+error[E0747]: constant provided when a type was expected
```

Comment:
Is this error here ok? Seems to be different

Reply:
it seems good to me

we have `Box<Bar>` where `Bar` is a function object :thinking: 
what do you think is incorrect here?

Reply:
ah sorry, I didn't look at the original file, but since the previous error didn't report a missing type it seemed different. Looking at the original file it seems fine now

Reply:
We previously emitted 2 errors here and only the second one mentions types
```
error[E0107]: wrong number of const arguments: expected 0, found 1
  --> $DIR/privacy-ns2.rs:41:18
   |
LL |     let _x : Box<Bar>;
   |                  ^^^ unexpected const argument

error[E0107]: wrong number of type arguments: expected at least 1, found 0
  --> $DIR/privacy-ns2.rs:41:14
   |
LL |     let _x : Box<Bar>;
   |              ^^^^^^^^ expected at least 1 type argument
```

Reply:
Ah ok, should be totally fine then, I'm just blind